src/gar.lua

```diff
@@ -855,4 +870,48 @@ function gar.instantDelegateWithdrawal(from, gatewayAddress, vaultId, currentTim
 	}
 end
 
+function gar.instantOperatorWithdrawal(from, vaultId, currentTimestamp)
+	local gateway = gar.getGateway(from)
+
+	if gateway == nil then
+		error("Gateway not found")
+	end
+
+	local vault = gateway.vaults[vaultId]
+	if vault == nil then
+		error("Vault not found")
+	end
+
+	if vaultId == from then
+		error("This gateway is leaving and this vault cannot be instantly withdrawn.")
+	end
+
+	-- Calculate elapsed time since the withdrawal started
+	local elapsedTime = currentTimestamp - vault.startTimestamp
+	local totalWithdrawalTime = gar.getSettings().operators.withdrawLengthMs
+
+	-- Ensure the elapsed time is not negative
+	if elapsedTime < 0 then
+		error("Invalid elapsed time")
+	end
+
+	-- Process the instant withdrawal
+	local expeditedWithdrawalFee, amountToWithdraw =
```

Comment:
Need to collect the penaltyRate here to be used down below.

Reply:
Doh... fixed.

https://github.com/ar-io/ar-io-network-process/commit/16d793cf9182e98d8b3a316030a48f97c3fc6242